### ISSUE TITLE
Unable to create new users - Helm installation

### ISSUE THREAD

**Issue by abhi5981** [ArkCase/ark_helm_charts#52]
Hi,

I installed arkcase in my kubernetes cluster using helm with the help of https://arkcase.github.io/ark_helm_charts/

I installed the community version, since I want to use it as open source, by disabling the license data.

It is successfully installed and I am able to login with Username: arkcase-admin@dev.arkcase.com and Password: $arkcase-admin$

Now, I want to create other users as well who could login with their username and password.

I am using below to create user:

Create New User(s)
Login to the application as the admin user
Go to the Admin Module
Select Security > Organizational Hierarchy
Select a group and click ‘Add New Member’
ARKCASE_ADMINISTRATOR – ARKCASE_SUPERVISOR access and system administration
ARKCASE_SUPERVISOR – ARKCASE_CONTRIBUTOR access and task approval
ARKCASE_CONTRIBUTOR – able to collaborate with other users.
ARKCASE_CONSUMER – read access to modules and information.
Enter the user’s information and click ‘Add LDAP User’

Also in table "acm_user" these users are getting updated.

But the issue is that what is the password for these users that I am creating, because nowhere I am getting option to set password for these users while creating them.

Need help this urgently again.

**Comment by drivera-armedia** [comment]
Hi!

Each user should get an e-mail containing the password reset link. In order for e-mail routing to work, you must configure the e-mail settings as documented [here](https://arkcase.github.io/ark_helm_charts/docs/External_Services.html#email). Pay special attention to the `connect` value, and if your relay is unauthenticated, simply set the `username` and `password` values to the empty string (i.e. `""`).

Cheers!

**Comment by abhi5981** [comment]
Hi,
Will set this up and confirm if it works. 
Thanks.

**Comment by abhi5981** [comment]
Hi, 

I set up the smtp, email sending is failing. Unable to reset passwords for user. Attaching logs here.

Tested the SMTP server separately( separately from Arkcase ) and its working fine. 
[smtp_logs_github.txt](https://github.com/user-attachments/files/17747571/smtp_logs_github.txt)

Please check this.

**Comment by abhi5981** [comment]
Hi,

Did you get a chance to have a look at this ?

**Comment by drivera-armedia** [comment]
Yes. The SMTP host _cannot_ be `localhost` because the SMTP relay is not running inside the pod itself. You'll need to use the actual network name or IP for the SMTP relay.
